### PR TITLE
New awareness questions

### PR DESCRIPTION
**Pull request by brurez**
This PR makes the app pop up a modal with a new question when the user clicks the "thumbs up" button. This question is a new awareness question to make the user give a second thought about the reasons it's buying the product.

### CODE REVIEW

File: src/Components/Home/questions.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[
+  "How often do you think you are going to use this product?",
+  "Do you have free space to store this product when you are not using it?",
+  "Is this product you are considering buying safe for the environment? Why?"
+]
```
Comment (dmytrokulak) [inline src/Components/Home/questions.json:5]: I think the second question would provoke more reflection if ask "Where do you have free space...?"  The third one is really two questions, let's make it simpler and also make it open-ended: "How can buying this product be harmful to environment?" I feel like "yes/no" answers doesn't really make a user to think over the problem.

We can also add "which alternative products have you considered?"

File: src/Components/Home/NewQuestionModal.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+          variant="primary"
+          onClick={() =>
+            props.onSaveClick({
+              productId: props.productId,
+              question: props.question,
+              answer: props.answer,
+            })
+          }
+        >
+          Save and close
```
Comment (dmytrokulak) [inline src/Components/Home/NewQuestionModal.js:36]: what if we have two buttons: one would add to likes, others not? If user has to agree "yeah, I don't have space" - why would s/he get a point to likes? this change is going to mean more development work but feels more logical, imho. Now a user can get a like incremented even if the field is left empty.

what about "save and close" be smt like "Still want to buy" and another one be ... "should reconsider buying"?

Comment (dmytrokulak) [reply src/Components/Home/NewQuestionModal.js]: I'll move this to a backlog card, I feel like this needs discussion

File: src/Components/Home/NewQuestionModal.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Modal, Form, Button } from "react-bootstrap";
+
+const NewQuestionModal = (props) => {
+  return (
+    <Modal show={props.show} onHide={props.onHide}>
+      <Modal.Header>
+        <Modal.Title>New Question</Modal.Title>
```
Comment (dmytrokulak) [inline src/Components/Home/NewQuestionModal.js:8]: I'm not sure about the title. It attracts attention but doesn't provide any new useful information.

**Review by dmytrokulak — COMMENTED** [review]
When all questions are used up, a pop is still available and a user can still gain more likes for a question 
image

File: src/Components/Home/NewQuestionModal.js (hunk 3 above)
Comment (brurez) [reply src/Components/Home/NewQuestionModal.js]: I changed in the last commit.

Comment (brurez) [reply src/Components/Home/NewQuestionModal.js]: Now it allows the user to increase the product score even when there are no more questions to answer.

**Comment by dmytrokulak** [comment]
Let's merge it when checks are good.

**Review by dmytrokulak — APPROVED** [review]

**Pull request merged**